src/content/docs/en/guides/cms/tina-cms.mdx

```diff
@@ -143,3 +143,11 @@ To get started, you'll need an existing Astro project.
     Learn more about Tina collections [in the Tina docs](https://tina.io/docs/reference/collections/).
 
 5. In production, TinaCMS can commit changes directly to your GitHub repository. To set up TinaCMS for production, you can choose to use [Tina Cloud](https://tina.io/docs/tina-cloud/) or self-host the [Tina Data Layer](https://tina.io/docs/self-hosted/overview/). You can [read more about registering for Tina Cloud](https://app.tina.io/register) in the Tina Docs.
+
+## Official Resources
+
+- TinaCMS provides their own [Astro integration guide](https://www.storyblok.com/mp/announcing-storyblok-astro](https://tina.io/docs/frameworks/astro/).
```

Comment:
```suggestion
- [TinaCMS Astro integration guide](https://tina.io/docs/frameworks/astro/).
```
Just tidied this up! We try to make sure that the link has a lot of context within it it (e.g. the equivalent of avoiding "here" or "click here") so maing sure that TinaCMS is also in the linked text is helpful to distinguish and inform exactly where this link is going!